test/topicFilters.spec.ts

```diff
@@ -431,3 +441,190 @@ describe("Send/Receive messages using boolean filters of subscription", function
     await testPeekMsgsLength(subscriptionClient, 0);
   });
 });
+
+describe("Send/Receive messages using sql filters of subscription", function(): void {
+  beforeEach(async () => {
+    await beforeEachTest();
+  });
+
+  afterEach(async () => {
+    await receiveOrders(defaultSubscriptionClient);
+    await testPeekMsgsLength(defaultSubscriptionClient, 0);
+
+    await afterEachTest();
+  });
+
+  it("SQL rule filter on the message properties", async function(): Promise<void> {
+    await subscriptionClient.addRule("SQLMsgPropertyRule", "sys.label = 'red'");
+
+    const rules = await subscriptionClient.getRules();
+    should.equal(rules.length, 1, "rule");
+    should.equal(rules[0].name, "SQLMsgPropertyRule");
```

Comment:
We are adding new rules to the subscription in each test case. They all get removed before started a new test, but by the end, we are left with the rule added by the last test case.

We should update the `afterEach` section to remove all the filters and then add the default filter back. Because, in the absence of any rules, the subscription wont get any messages. This can fail tests in the other files